people_recognition_3d/src/people_recognition_3d/people_recognizer_3d.py

```diff
@@ -257,7 +257,9 @@ def recognize(self, rgb, depth, camera_info):
                       (rospy.Time.now() - t).to_sec())
         rospy.loginfo('Found {} people'.format(len(people2d)))
 
-        cmap = color_map(N=len(people2d), normalized=True)
+        max_group = max(person2d.body_parts[0].group_id for person2d in people2d) if people2d else 0
+        rospy.loginfo("max_group = {}".format(max_group))
+        cmap = color_map(N=max_group+1, normalized=True)
```

Comment:
Why `max_group+1`?

Reply:
Explained in the comment as the group_id starts from `1` instead of `0`. The color map is being queried based on the group_id. The `+1` can be removed, then the query index must change to `group_id - 1`.